product-scenarios/scenarios-common/src/main/java/org/wso2/am/scenario/test/common/ScenarioTestBase.java

```diff
@@ -81,6 +82,7 @@ protected static void setup() {
             storeURL = "https://localhost:9443/store";
         }
         setKeyStoreProperties();
+        httpsAPIInvocationURL = "https://localhost:8243";
```

Comment:
TG has  a URL specified for this in the name of "GatewayHttpsUrl" shall we use this? just like we are resolving publisherUrl, resolve this URL as well

Reply:
Resolved